uber/config.py

```diff
@@ -344,4 +357,4 @@ def _is_intstr(s):
 
 c.SAME_NUMBER_REPEATED = r'^(\d)\1+$'
 
-stripe.api_key = c.STRIPE_SECRET_KEY
+stripe.api_key = c.STRIPE_SECRET_KEY
```

Comment:
Can you configure PyCharm to add newlines at the end of the file?  My editor does that (which I personally prefer anyway), and if we're not in sync on that then every time oneof us edits a file the other touched last we'll get a pointless line in the diff like this.